### PR TITLE
Dashboard Code base

### PR DESCRIPTION
**Pull request by MVarshini**
CHAOS-608

Dashboard Code base

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+app.use(cors());
+app.use(express.json());
+
+app.post("/start-kraken/", (req, res) => {
+  const passwd = req.headers?.authorization?.split(" ")[1];
+  const scenario = req.body.params.scenarioChecked;
+  let command = "";
+  switch (scenario) {
+    case "pod-scenarios":
+      command = `echo '${passwd}' | sudo -S podman run --env NAMESPACE=${req.body.params.namespace} --env NAME_PATTERN=${req.body.params.name_pattern} --env POD_LABEL=${req.body.params.pod_label} --env DISRUPTION_COUNT=${req.body.params.disruption_count}  --env KILL_TIMEOUT=${req.body.params.kill_timeout} --env WAIT_TIMEOUT=${req.body.params.wait_timeout} --env EXPECTED_POD_COUNT=${req.body.params.expected_pod_count} --name=ui --net=host --env-host -v ${req.body.params.kubeconfigPath}:/root/.kube/config:Z -d quay.io/redhat-chaos/krkn-hub:pod-scenarios`;
```
Comment (chaitanyaenr) [inline server/index.js:20]: Maybe we should add checks to make sure podman is installed? Thoughts?

Comment (MVarshini) [reply server/index.js:20]: Okay. Will work on it

Comment (MVarshini) [reply server/index.js:20]: Included the check for podman status

**Review by chaitanyaenr — APPROVED** [review]
LGTM!

**Pull request merged**